src/test/java/org/assertj/core/error/DescriptionFormatter_format_Test.java

```diff
@@ -46,15 +46,20 @@ public void should_format_description_if_value_is_not_empty_or_null() {
   }
 
   @Test
-  @Parameters(method = "testDescriptionGenerator")
+  @UseDataProvider("testDescriptionGenerator")
   public void should_return_empty_String(TestDescription testDescription) {
     assertThat(formatter.format(testDescription)).isEmpty();
   }
 
-  @SuppressWarnings("unused")
-  private Object testDescriptionGenerator() {
-    return array(null,
-                 new TestDescription(null),
-                 new TestDescription(""));
+  @DataProvider(format="%m[%i]") // Workaround with custom format because TestDescription#toString returns the value
```

Comment:
I will create a pull request based on the feedback I receive at https://github.com/TNG/junit-dataprovider/issues/66